docs/docs/documentation/getting-started/features.md

```diff
@@ -164,6 +163,53 @@ Managing a robust collection of recipes inevitable requires a lot of data. Meali
 
 [Data Management Demo](https://demo.mealie.io/group/data/foods){ .md-button .md-button--primary }
 
+## Groups and Households
+
+Mealie lets you fully customize how you organize your users. You can use Groups to host multiple instances (or tenants) of Mealie which are completely isolated from each other. Within each Group you can organize users into Households which allow users to share recipes, but keep other items separate (e.g. meal plans and shopping lists).
+
+### Groups
+
+Groups are fully isolated instances of Mealie. Think of a goup as a completely separate, fully self-contained site. There is no data shared between groups. Each group has its own users, recipes, tags, categories, etc. A user logged-in to one group cannot make any changes to another.
+
+Common use cases for groups include:
+- Hosting multiple instances of Mealie for others who want to keep their data private and secure
+- Creating completely isolated recipe pools
+
+### Households
+
+Households are subdivisions within a single Group. Households maintain their own users and settings, while sharing their recipes with other households. Households also share organizers (tags, categories, etc.) with the entire group. Meal Plans, Shopping Lists, and Integrations are only accessible within a household.
+
+Common use cases for households include:
+- Sharing a common recipe pool amongst families
+- Maintaining separate meal plans and shopping lists from other households
+- Maintaining separate integrations and customizations from other households
+
+```mermaid
+flowchart TB
+    mealie[(Mealie)]:::primary ==> groups
+
+    %% Groups
+    groups((Groups)):::secondary --> ingredients & organizers
+    groups((Groups)):::secondary ====> households
+    ingredients("Ingredients<br/>(Foods, Units, Labels)")
+    organizers("Organizers<br/>(Categories, Tags, Tools)")
+
+    %% Households
+    households((Households)):::secondary --> recipes & mealplans & shoppinglists & integrations
+
+    recipes(Recipes & Cookbooks)
+    mealplans(Meal Plans)
+    shoppinglists(Shopping Lists)
+    integrations("Integrations<br/>(Notifiers, Webhooks)")
+
+    %% Styling
+    classDef primary fill:#E58325,stroke:#B4B7C2,color:#FFFFFF
+    classDef secondary fill:#1976D2,stroke:#B4B7C2,color:#FFFFFF
+
+    classDef default fill:#1F2332,stroke:#E58325,color:#B4B7C2
+    linkStyle default stroke:#B4B7C2
+```
```

Comment:
Seems like text color is not supported this way in mkdocs. This is  kinda fine for the Mealie, Groups and Households. 
But the contrast with the background color for the other nodes makes them almost illegible.

Reply:
Huh, weird. Sure I can just get rid of the styling and let mkdocs do its thing

Reply:
Yeah, was wondering the same. 
They state say that color does not work for all diagrams but they explicitly mention that it works for flowcharts so i don't know whats up with that 🤷‍♂️